applications/ParticleMechanicsApplication/python_scripts/deprecated_python_scripts/deprecated_particle_mpm_solver.py

```diff
@@ -0,0 +1,370 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications and dependencies
+import KratosMultiphysics.SolidMechanicsApplication as KratosSolid
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
+
+# Check that KratosMultiphysics was imported in the main script
+KratosMultiphysics.CheckForPreviousImport()
```

Comment:
this function is currently not working, please remove it

Reply:
this function is deprecated Philipp, should I modify still?